docs/Training-on-Microsoft-Azure.md

```diff
@@ -1,4 +1,4 @@
-# Training on Microsoft Azure
+# Training on Microsoft Azure (works with ML-Agents v0.3)
```

Comment:
Doesn't it work with 0.4 ?

Reply:
Azure documentation doesn't work with 0.4 for now, because in their documentation they are still using tensorflow 1.4 and cuda 8.0. I've asked them to update it, but hasn't seen the corresponding PR yet.